reconcile/test/test_utils_gql.py

```diff
@@ -0,0 +1,83 @@
+import requests
+
+from reconcile.utils.gql import (
+    GqlApi,
+    GqlApiError,
+    GqlApiErrorForbiddenSchema,
+    GqlApiIntegrationNotFound,
+)
+from gql import Client
+from gql.transport.exceptions import TransportQueryError
+import pytest
+
+TEST_QUERY = """
+{
+    integrations: integrations_v1 {
+        name
+        description
+        schemas
+    }
+}
+"""
+
+
+@pytest.fixture
+def mocked_client(mocker):
+    mocked_client = mocker.Mock(spec=Client, create_autospec=True)
```

Comment:
Is `create_autospec` a valid kwarg for `Mock`? Check out: https://docs.python.org/3/library/unittest.mock.html

Reply:
Good catch! I was blind when I looked at plugin.py for mocker, that had create_autopec.

Reply:
The last bit is that `spec` is a positional arg in `create_autospec`, so you don't even need the `spec=` part.